scripts/Muon/GUI/ElementalAnalysis/LoadWidget/load_utils.py

```diff
@@ -48,8 +49,16 @@ def load_run(self):
         for ws_name in workspaces.values():
             loaded_detectors[ws_name[0]] = 1
         num_loaded_detectors = len(loaded_detectors)
+
+        if (mantid.AnalysisDataService.Instance().doesExist(str(self.run))):
+            detector_list = get_detector_list(self.run, workspaces.values())
```

Comment:
instead of using this it would be better to use the group workspace to get its members.